.changeset/many-tables-appear.md

```diff
@@ -0,0 +1,18 @@
+---
+'merchant-center-application-template-starter': major
+'@commercetools-frontend/mc-dev-authentication': major
+'@commercetools-frontend/mc-html-template': major
+'@commercetools-frontend/mc-scripts': major
+'playground': major
+'@commercetools-local/visual-testing-app': major
+---
+
+Following breaking changes were introduced:
+
+- In `mc-scripts`, the `build` command additionally compiles the `index.html` by default.
+- Running the `compile-html` command by default should not be necessary anymore. However, you can pass `--build-only` to the `build` command to opt-out of the compilation step, in case you want to run it separately, for example to use the `--transformer`.
```

Comment:
Maybe me but I rad this four times until I got it 😄 

```suggestion
- Running `build` now includes compiling the HTML. As a result running the `compile-html` command should not be necessary anymore. However, if you do not need HTML to be compiled with the `build` command pass the `--build-only` to opt-out of the compilation step. A use case for this is when using a `--transformer` to run compilation separately.
```

Reply:
Thanks :)